pkg/common/common.go

```diff
@@ -3,25 +3,58 @@ package common
 import (
 	"fmt"
 
+	"github.com/vmware-tanzu/velero/pkg/restore"
 	corev1 "k8s.io/api/core/v1"
 )
 
 const (
-	Velero                           = "velero"
-	NodeAgent                        = "node-agent"
-	VeleroNamespace                  = "oadp-operator"
-	OADPOperator                     = "oadp-operator"
-	OADPOperatorVelero               = "oadp-operator-velero"
-	DataMover                        = "volume-snapshot-mover"
-	DataMoverController              = "data-mover-controller"
-	DataMoverControllerContainer     = "data-mover-controller-container"
-	OADPOperatorServiceAccount       = "openshift-adp-controller-manager"
-	VolSyncDeploymentName            = "volsync-controller-manager"
-	VolSyncDeploymentNamespace       = "openshift-operators"
-	VolsyncPrivilegedAnnotation      = "volsync.backube/privileged-movers"
-	DefaultRestoreResourcePriorities = "securitycontextconstraints,customresourcedefinitions,namespaces,storageclasses,volumesnapshotbackups.datamover.oadp.openshift.io,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io"
+	Velero                       = "velero"
+	NodeAgent                    = "node-agent"
+	VeleroNamespace              = "oadp-operator"
+	OADPOperator                 = "oadp-operator"
+	OADPOperatorVelero           = "oadp-operator-velero"
+	DataMover                    = "volume-snapshot-mover"
+	DataMoverController          = "data-mover-controller"
+	DataMoverControllerContainer = "data-mover-controller-container"
+	OADPOperatorServiceAccount   = "openshift-adp-controller-manager"
+	VolSyncDeploymentName        = "volsync-controller-manager"
+	VolSyncDeploymentNamespace   = "openshift-operators"
+	VolsyncPrivilegedAnnotation  = "volsync.backube/privileged-movers"
 )
 
+var DefaultRestoreResourcePriorities = restore.Priorities{
+	HighPriorities: []string{
```

Comment:
niceeeee :rocket: much better to read

Reply:
@mateusoliveira43 yes -- upstream provides the restore.Priorities struct. I realized that I should just use that struct and let it convert to string with the `.String()` method provided. Still need the messy list in test expectations, cli docs, etc. though

Reply:
yeah, if we could dynamically get that in docs (CRD description) would be perfect

Reply:
@mateusoliveira43 We'd probably have to do that with a make task somehow, since it's in a golang comment, we can't exactly run golang code there. But yeah, something to think about at some point.